lib/Customer.php

```diff
@@ -2,6 +2,28 @@
 
 namespace Stripe;
 
+/**
+ * Class Customer
+ *
+ * @property string $id
+ * @property string $object
+ * @property int $account_balance
+ * @property string $business_vat_id
+ * @property string $created
+ * @property string $currency
+ * @property string $default_source
+ * @property bool $delinquent
+ * @property string $description
+ * @property mixed $discount
+ * @property string $email
+ * @property bool $livemode
+ * @property array $metadata
+ * @property mixed $shipping
+ * @property array $sources
+ * @property array $subscriptions
```

Comment:
I believe `$sources` and `$subscriptions` are of type `\Stripe\Collection`. Using the collection class instead of `array`, would allow IDEs to recommend the methods that we can use on those two properties. Like `retrieve`, `create`, etc.